salt/modules/win_snmp.py

```diff
@@ -340,6 +412,11 @@ def set_community_names(communities):
     '''
     values = dict()
 
+    if __salt__['reg.key_exists'](_HKEY, _COMMUNITIES_GPO_KEY):
+        _LOG.debug('Communities on this system are managed by Group Policy')
+        raise CommandExecutionError(
```

Comment:
I agree it is better to `raise` here, but I think it is at least worth a release note that this is a change in behavior (if not a deprecation path). Someone may be using this function on a system now where Group Policy happens to be managing the communities  Salt would succeed previously (if a little pointlessly), and this change will cause a bit of breakage when they update.

Reply:
The state wouldn't fail, as it checks for the CommandExecutionError. Only a Command line run or a salt api run on the execution module would fail.

Reply:
Right... The point remains, yes? The behavior is changing in a way that breaks prior usage of this function. Certainly salt states are designed to handle that, but salt states are not the sole user of execution modules.

Reply:
Right. I added this to the release notes.